projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -202,6 +203,12 @@ public function admin_styles() {
 		wp_style_add_data( 'jetpack-admin', 'rtl', 'replace' );
 		wp_style_add_data( 'jetpack-admin', 'suffix', $min );
 	}
+	/**
+	 * Adds iOS Smart banner to let folks know about the Jetpack app on iOS.
+	 */
+	public function add_ios_smart_banner() {
```

Comment:
This feels like the wrong place to add that method, since the meta tag is added to every page in wp-admin.

What do you think about creating a new file loaded in `projects/plugins/jetpack/3rd-party/3rd-party.php` where we would add this, and potentially other things in the future?

I would also recommend adding a check to ensure this only gets added once, because in my tests the meta tag often got added 3 times.